apps/admin_app/mix.exs

```diff
@@ -40,6 +40,7 @@ defmodule AdminApp.Mixfile do
       {:phoenix_pubsub, "~> 1.0"},
       {:phoenix_html, "~> 2.10"},
       {:phoenix_live_reload, "~> 1.0", only: :dev},
+      {:phoenix_ecto, "~> 3.0"},
```

Comment:
Why was this added? The `admin_app` has no `Ecto.Repo`.

Reply:
for using changesets in forms otherwise forms won't be able to convert changeset data.